projects/canopy/src/lib/paginator/paginator.component.html

```diff
@@ -0,0 +1,52 @@
+<lg-select-field class="lg-paginator__page-size" *ngIf="!hidePageSize">
```

Comment:
I'm wondering if the page size select field should be a separate component? I guess it depends on design somewhat, but could there be a scenerio where we'd want to show this selection at the top of the page and the actual pagination buttons at the bottom? Maybe it's worth waiting to see what the designs are and what the actual use-case is within our app.